### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by divyanshu-rawat**
Bug fixes to make defaultSelected story to work.

### CODE REVIEW

File: packages/vue/src/components/range/RangeSlider.jsx [hunk 1]
```diff
@@ -129,14 +130,22 @@ const RangeSlider = {
...
+
     this.addComponent(this.$props.componentId);
+    this.addComponent(this.internalComponent);
     this.setReact(this.$props);
-    if (this.selectedValue) {
-      this.handleChange(this.selectedValue);
-    } else if (this.$props.defaultSelected) {
-      this.handleChange(this.$props.defaultSelected);
+
+    const { selectedValue, defaultSelected } = this.$props;
```
Comment (bietkul) [inline packages/vue/src/components/range/RangeSlider.jsx]: How you're accessing this `selectedValue` from `this.$props`, afaik it's coming from redux.

Comment (divyanshu-rawat) [reply packages/vue/src/components/range/RangeSlider.jsx]: @bietkul yes it is coming as a prop from here - 
```
  selectedValue: state.selectedValues[props.componentId]
    ? state.selectedValues[props.componentId].value
    : null,
```

**Comment by divyanshu-rawat** [comment]
story

File: packages/vue/src/components/range/RangeSlider.jsx (hunk 1 above)
Comment (divyanshu-rawat) [reply packages/vue/src/components/range/RangeSlider.jsx]: Updated

File: packages/vue/src/components/range/RangeSlider.jsx [hunk 2]
```diff
@@ -131,14 +130,23 @@ const RangeSlider = {
     this.setQueryListener(this.$props.componentId, onQueryChange, null);
   },
   beforeMount(){
+
     this.addComponent(this.$props.componentId);
+    this.addComponent(this.internalComponent);
```
Comment (jyash97) [inline packages/vue/src/components/range/RangeSlider.jsx]: What is `this.internalComponent` doing? It is not initialized

Comment (divyanshu-rawat) [reply packages/vue/src/components/range/RangeSlider.jsx]: Sure,then you can git rid of it.

**Pull request merged**